packages/mds-ingest-service/tests/index.spec.ts

```diff
@@ -102,20 +103,57 @@ const TEST_TELEMETRY_B2: TelemetryDomainCreateModel = {
   timestamp: testTimestamp + 1000
 }
 
-const TEST_TELEMETRY_ANNOTATION_A1: TelemetryAnnotationDomainCreateModel = {
+const BASE_TEST_TELEMETRY_ANNOTATION = {
   device_id: DEVICE_UUID_A,
   provider_id: TEST1_PROVIDER_ID,
-  timestamp: testTimestamp,
   h3_08: '123451234512345',
   h3_09: '123451234512345',
   h3_10: '123451234512345',
   h3_11: '123451234512345',
   h3_12: '123451234512345',
   h3_13: '123451234512345',
-  telemetry_row_id: 12341234,
   geography_ids: []
 }
 
+const TEST_TELEMETRY_ANNOTATION_A1: TelemetryAnnotationDomainCreateModel = {
+  timestamp: testTimestamp,
+  telemetry_row_id: 12341234,
+  ...BASE_TEST_TELEMETRY_ANNOTATION
+}
+
+const TEST_TELEMETRY_ANNOTATION_A2: TelemetryAnnotationDomainCreateModel = {
+  timestamp: testTimestamp + 1000,
+  telemetry_row_id: 12341235,
+  ...BASE_TEST_TELEMETRY_ANNOTATION
+}
+
+const TEST_TELEMETRY_ANNOTATION_A3: TelemetryAnnotationDomainCreateModel = {
+  timestamp: testTimestamp + 2000,
+  telemetry_row_id: 12341236,
+  ...BASE_TEST_TELEMETRY_ANNOTATION
+}
+
+const TEST_TELEMETRY_ANNOTATION_A4: TelemetryAnnotationDomainCreateModel = {
+  timestamp: testTimestamp + 3000,
+  telemetry_row_id: 12341237,
+  ...BASE_TEST_TELEMETRY_ANNOTATION,
+  h3_09: 'imunlikeother09'
+}
+
+const TEST_TELEMETRY_ANNOTATION_A5: TelemetryAnnotationDomainCreateModel = {
+  timestamp: testTimestamp + 4000,
+  telemetry_row_id: 12341238,
+  ...BASE_TEST_TELEMETRY_ANNOTATION,
+  h3_09: 'IMUNLIKEOTHER09'
+}
+
+const TEST_TELEMETRY_ANNOTATION_A6: TelemetryAnnotationDomainCreateModel = {
+  timestamp: testTimestamp + 5000,
+  telemetry_row_id: 12341239,
+  ...BASE_TEST_TELEMETRY_ANNOTATION,
+  h3_09: 'IMUNLIKEOTHER09'
+}
```

Comment:
Feels like a ripe candidate for a factory function & generator versus explicitly defining `n` things